### ISSUE TITLE
Possible typos in chapters 6, 7 and 9

### ISSUE THREAD

**Issue by fayolle** [shokru/mlfactor.github.io#61]
Some possible typos spotted while reading chapters 6 to 9: 
- trees.html, section 6.4.5
"In Figure ," -> "In Figure (6.6)," 
http://www.mlfactor.com/trees.html#boosted-trees-extreme-gradient-boosting

- NN.html
  - Section 7.3.3: \delta_j instead of \delta_k in the expression of O 
  - In 7.4.2 (second to last paragraph): “ validation accuracy (“val_acc”; one alternative is “val_loss”).” -> It is the opposite in the code. 
  - In the "solutions-to-exercises.html" chapter, it seems that the numbering for the solutions is off by one. For example, the solution to the neural network exercise (chapter 7) is listed in "Chapter 8: the autoencoder model". 

- bayes.html
  - In 9.2 "generator is symmetric (p(x|y) = (y|x))" -> p(x|y) = p(y|x)? 
  - In 9.5 "The performance seems reasonable but is by no means not impressive." -> "is by no means impressive"?

**Comment by shokru** [comment]
Thank you again for your very thorough reading. 
I thus have added you in the acknowledgments :)
And the book is now updated.

**Comment by fayolle** [comment]
Thank you. That's very kind of you.
